<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeformMesh</class>
 <widget class="QDockWidget" name="DeformMesh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Surface Mesh Deformation</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="SelectionGroupBox">
      <property name="title">
       <string>Selection</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="1" rowspan="2" colspan="3">
        <layout class="QVBoxLayout" name="RingLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Brush Size ROI:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Control Vertices Brush Size:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QSpinBox" name="BrushSpinBoxRoi"/>
             </item>
             <item>
              <widget class="QSpinBox" name="BrushSpinBoxCtrlVert"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="5">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="SelectAllVerticesPushButton">
           <property name="text">
            <string>Set All Vertices as ROI</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ClearROIPushButton">
           <property name="text">
            <string>Clear ROI</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" rowspan="2">
        <widget class="QGroupBox" name="ROICtrlVertGroupBox">
         <layout class="QVBoxLayout" name="CtrlVertRoiLayout">
          <item>
           <widget class="QRadioButton" name="ROIRadioButton">
            <property name="toolTip">
             <string>Use Shift + Left Click to paint ROI vertices</string>
            </property>
            <property name="text">
             <string>ROI</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="CtrlVertRadioButton">
            <property name="toolTip">
             <string>Use Shift + Left Click to paint control vertices</string>
            </property>
            <property name="text">
             <string>Control vertices</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="4" rowspan="2">
        <widget class="QGroupBox" name="InsertEraseGroupBox">
         <layout class="QVBoxLayout" name="InsertRemoveLayout">
          <item>
           <widget class="QRadioButton" name="InsertRadioButton">
            <property name="text">
             <string>Insertion</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="EraseRadioButton">
            <property name="text">
             <string>Removal</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Isolated Component Size:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="Threshold_size_spin_box">
             <property name="maximum">
              <number>999999999</number>
             </property>
             <property name="value">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Get_minimum_button">
             <property name="text">
              <string>Get Minimum</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="Select_isolated_components_button">
           <property name="text">
            <string>Select Isolated Components Below Threshold</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="CtrlVertGroupNavigationGroupBox">
      <property name="title">
       <string>Group of Control Vertices Navigation</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="PrevCtrlVertPushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&lt;&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="NextCtrlVertPushButton">
           <property name="text">
            <string>&gt;&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="AddCtrlVertPushButton">
           <property name="text">
            <string>Create new</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="DeleteCtrlVertPushButton">
           <property name="text">
            <string>Delete </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="ActivatePivotingCheckBox">
         <property name="text">
          <string>Activate Pivoting </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="OverwritePushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Update Original Positions</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QCheckBox" name="ShowROICheckBox">
        <property name="text">
         <string>Show ROI</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ShowAsSphereCheckBox">
        <property name="text">
         <string>Show As Sphere</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="SaveROIPushButton">
        <property name="text">
         <string>Save ROI / Control Vertices</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ReadROIPushButton">
        <property name="text">
         <string>Load ROI / Control Vertices</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="ApplyAndCloseLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ApplyAndClosePushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Apply and Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
